debian/install

```diff
@@ -1 +1,2 @@
 ubports-qa /usr/bin
+tools/completions/ubports-qa usr/share/bash-completion/completions/ubports-qa
```

Comment:
Remove the filename `ubports-qa` from the target path `usr/share/bash-completion/completions/ubports-qa`. I think it should work then ...